### DISCUSSION TITLE
Copilot Questions...

### DISCUSSION THREAD

**Discussion by Shai69bb** [community/community#42009] [Copilot Conversations]
Hello.
Is it possible to get my Github Copilot account synced on my phone and laptop for the same account?

**Comment by SilasRodrigues19** [comment]
Hi @Shai69bb 

Yes, it is possible to sync your GitHub Co-Pilot account between different devices, such as a phone and a laptop. This is done by setting up the account on each device and logging in with the same credentials on each one.

To set up your GitHub Co-Pilot account on a new device, you will need to do the following:

1. Download the GitHub Co-Pilot app for the device you want to set up.
2. Open the app and click "Sign In" on the welcome screen.
3. Enter your GitHub username and password and click "Sign In".
4. If you have multiple GitHub accounts, select the account you want to use to access GitHub Co-Pilot.
5. After successfully logging in, your GitHub Co-Pilot account will be synced with the device. Any changes you make on one device will be reflected on all other devices where you have set up your account.

Note that for the synchronization to work properly, it is important that you are connected to the internet on all devices where you want to sync your account. If you are working on a device without a connection, changes will only be synced when you reconnect to the internet.

**Reply by Shai69bb** [reply]
Thanks for the detailed response to my message. Really appreciate your help.

**Comment by slim-python** [comment]
i have a doubt.

i have two github account (one is for personal one is for office, office one uses bitbucket)
if i buy the github co -pilot from my personal github account, will i be able to use in both the accounts?

**Reply by Shai69bb** [reply]
Hi yes you can. I'm attaching the answer I received for pretty much the same question.[
![Screenshot_20230220_203937_Chrome](https://user-images.githubusercontent.com/58149966/220226121-927234c3-f965-4c09-8355-03f716d3bac8.jpg)
](url)

**Comment by ollyde** [comment]
Is it possible for co-pilot to read the whole project in order to learn it? I noticed that it only 'learns' what I've dealt with so far, which is kinda annoying.
